public/services/saved_objects/saved_object_savers/ppl/save_as_current_vis.ts

```diff
@@ -46,9 +46,12 @@ export class SaveAsCurrentVisualization extends SavedQuerySaver {
   }
 
   addToPanel({ selectedPanels, saveTitle, notifications, visId }) {
+    const uuidRx = /^[0-9a-fA-F]{8}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{12}$/;
+    const soPanels = selectedPanels.filter((id) => id.panel.id.match(uuidRx));
+    const opsPanels = selectedPanels.filter((id) => !id.panel.id.match(uuidRx));
```

Comment:
- use `uuidRx.test(id.panel.id)`
- `id` variable name is not accurate
- i'm not sure if the logic should be here, e.g. what happens to panels in .observability index? `save_as_current_vis` don't support them anymore?
- not sure if it helps but you can check .observability and .kibana visualizations handling here as reference, for example bulk delete https://github.com/opensearch-project/dashboards-observability/blob/775a77ebda4ab64abdd20bab01841ccf94165edd/public/services/saved_objects/saved_object_client/saved_objects_actions.ts#L74-L88